go.mod

```diff
@@ -1349,6 +1350,9 @@ exclude (
 	github.com/knadh/koanf/providers/confmap v0.1.0
 )
 
+// todo: update datadog connector with breaking changes from https://github.com/DataDog/datadog-agent/pull/26347.
```

Comment:
cc @ajgajg1134 with the `replace` this PR shouldn't block  https://github.com/DataDog/datadog-agent/pull/26347, but please let us know if you ran into dependency issues.